tests/python/relay/test_graph_runtime.py

```diff
@@ -25,8 +25,10 @@ def check_rts(env, expr, args, expected_result):
     expected_result:
         The expected result of running the expression.
     """
-    eval_result = evaluate(env, expr, *args)
-    rts_result = graph_evaluate(env, expr, *args)
+    intrp = create_executor('graph', env=env)
```

Comment:
both are `graph`. Should one be `debug`?

Reply:
yeah I found this too. And ```expected_result```  (numpy result) is not used at all.